conbench/templates/app.html

```diff
@@ -136,8 +136,36 @@ <h4 class="modal-title"><span class="glyphicon glyphicon-trash submenu"></span>&
             table.search("{{ search_value }}").draw();
         {% endif %}
     });
+
 }
 
+// requires two inputs: table_id & column_indexes
+function specific_column_search(data_table, column_indexes){
+    $(document).ready(function() {
+        var table =  $(data_table).DataTable();
+        table.columns(column_indexes)
+		.every(function () {
+            var that = this;
+			var mySelectList = $("<select></select>")
```

Comment:
Looking at the dummy data, I can see that the Reason column has a "blank" choice, which might not have been intentional because some Reasons are actually blank in the dummy data. But I actually really like that behavior because it allows users to "deselect" the filter. Could you try to add a blank to every option list on purpose?

<img width="1182" alt="image" src="https://user-images.githubusercontent.com/16600275/212365469-29fe5873-faad-495a-b7fd-6c75f40d3203.png">

Reply:
I shall add a condition to identify empty/null value and then a separate reset feature. 
Else it will turn little confusing

Reply:
This looks great, thanks!